CHANGES/5470.bugfix

```diff
@@ -0,0 +1 @@
+Complete pip-tools setup.
```

Comment:
Urgh... This is now a bugfix but a misc because it's not a user-facing fix but something that would only affect the core devs and contributors.

Reply:
Sorry, my bad!

Reply:
It's not a big deal, I guess this can be fixed by a follow-up PR with `git mv`.